server/webdriver/shared.py

```diff
@@ -23,6 +23,11 @@
 ASYNC_ELEMENT_HOLDER_CLASS = 'dc-async-element-holder'
 ASYNC_ELEMENT_CLASS = 'dc-async-element'
 TIMEOUT = 60
+WEB_COMPONENT_TAG_NAMES = [
```

Comment:
Add a comment in here and the components definition page to sync both places?

Reply:
done!